tests/unit/cli/test_context.py

```diff
@@ -31,6 +34,39 @@ def test_must_unset_get_debug_flag(self):
         ctx.debug = False
         self.assertEqual(ctx.debug, False, "debug must be set to False")
 
+    def test_must_set_get_output_timestamp_flag(self):
+        ctx = Context()
+
+        message_record = logging.makeLogRecord({"msg": "hello world"})
+        timestamp_log_regex = re.compile(r"^[0-9:\- ,]+ \| .*$")
+
+        sam_cli_logger = logging.getLogger("samcli")
+        lambda_builders_logger = logging.getLogger("aws_lambda_builders")
+        SamCliLogger.configure_logger(sam_cli_logger, SAM_CLI_FORMATTER, logging.INFO)
+        SamCliLogger.configure_logger(lambda_builders_logger, SAM_CLI_FORMATTER, logging.INFO)
+
+        self.assertFalse(ctx.output_timestamp)
+        handlers = logging.getLogger("samcli").handlers + logging.getLogger("aws_lambda_builders").handlers
+        # timestamp should not be output
+        for handler in handlers:
+            self.assertNotRegex(handler.formatter.format(message_record), timestamp_log_regex)
+
+        ctx.output_timestamp = True
+        self.assertEqual(ctx.output_timestamp, True, "output_timestamp must be set to True")
+        # timestamp should be output
+        for handler in handlers:
+            self.assertRegexpMatches(handler.formatter.format(message_record), timestamp_log_regex)
+
+    def test_must_unset_get_output_timestamp_flag(self):
```

Comment:
Why do we need this test case?

Reply:
We do have one for the `--debug` flag, so this one is to keep it consistent with it
https://github.com/aws/aws-sam-cli/blob/f23baa9dc99ecb76ce12f88c14ffa48df39c983a/tests/unit/cli/test_context.py#L24-L32